src/net.c

```diff
@@ -276,4 +276,8 @@ void mg_mgr_init(struct mg_mgr *mgr) {
   mgr->dns4.url = "udp://8.8.8.8:53";
   mgr->dns6.url = "udp://[2001:4860:4860::8888]:53";
   mg_tls_ctx_init(mgr);
+
+#if (MG_ENABLE_DRIVER_INIT == 1) && defined(MG_TCPIP_DRIVER_INIT)
+  MG_TCPIP_DRIVER_INIT(mgr);
+#endif
```

Comment:
I think this does not belong here but in net_builtin.c
Here we can call that code when using our own stack, that is, here, we just call `mg_tcpip_init()` `#if MG_TCPIP_ENABLE`
`mg_tcpip_init` would take care to initialize the driver or not.

Reply:
Kind of agree with that, but how `net_builtin.c` would know about which driver to init ?
My thinking is that we already have `MG_ENABLE_DRIVER_xxx`, and so each of those respective drivers would #define an init macro in its header file - that's exactly what I did in this PR

Reply:
How does it know now ?
It gets a struct that has a pointer to the driver "methods" struct. That is included by having the proper driver enabled. Enabling more than one driver won't work at all, though.